### DISCUSSION TITLE
Rolladen Datenpunkt stop

### DISCUSSION THREAD

**Discussion by Bernd9000** [Zefau/ioBroker.jarvis#1559] [Q&A]
Wie kann ich in den Datenpunkt Eigenschaften des Datenpunkt "stop" bei einem Rolladen angeben das "stop" gesendet wird?

Hab es versucht mit {"stop":"stop"} {"default":"stop"} ect, aber es wird immer nur true gesendet.

Laut [jarvis-Infos](https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/geraet/gewerk/rolllaeden-blind-v3) ist das einstellbar ?
Zitat:
stop ist lediglich der Datenpunkt an den die Aktion geschickt wird. Hier wird kein Wert geladen, der die Visualisierung beeinflusst. Der Wert, der geschickt wird, kann über die Datenpunkt Eigenschaften via stop festgelegt werden. Im Standard (default) wird true geschickt.

**Comment by Zefau** [comment]
In den Datenpunkt-Eigenschaften `stop` ist korrekt.

**Reply by Bernd9000** [reply]
Wenn ich nur `stop` eintrage wird trotzdem nur `true` gesendet.
Ein bug?

**Comment by Bernd9000** [comment]
Issue angelegt

**Reply by Zefau** [reply]
https://github.com/Zefau/ioBroker.jarvis/issues/1560
